packages/wrangler/src/deployment-bundle/entry.ts

```diff
@@ -54,7 +54,7 @@ export async function getEntry(
 				? path.resolve(config.site?.["entry-point"])
 				: // site.entry-point could be a directory
 					path.resolve(config.site?.["entry-point"], "index.js");
-		} else if (args.assets || config.assets) {
+		} else if (args.legacyAssets || config.legacy_assets) {
```

Comment:
```suggestion
		} else if (args.legacyAssets || config.legacy_assets || config.assets) {
```
?

Reply:
nvm. rawConfig.assets and rawConfig.legacy_assets are merged and normalized into config.legacy_assets.